jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -308,8 +308,11 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/stg-config.json"
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/prod-config.json"
 		}
-
-	}
+		if (service_name.trim() == "jazz_test-lambda") {
```

Comment:
You will require to register this service as part of stack building: refer to the PR here for places where you need the changes: https://github.com/tmobile/jazz-installer/pull/219

Reply:
Added 
https://github.com/tmobile/jazz-installer/pull/222/files